omero/sysadmins/unix/server-centos6-ius.txt

```diff
@@ -0,0 +1,260 @@
+OMERO.server installation on Centos 6 with Python 2.7 and Ice 3.5
+=================================================================
+
+This installation walkthrough should be read in conjunction with
+:doc:`server-installation` and :doc:`install-web`.
+
+Running OMERO on CentOS 6 has a number of special requirements which
+deviate from the standard installation instructions.  The instructions
+below will set up Python 2.7 and Ice 3.5 on CentOS 6. We tested the
+installation with Python 2.7 from `IUS <https://ius.io/>`_ and used
+a `virtual environment <https://virtualenv.readthedocs.org/en/latest/>`_. 
+
+Install Python 2.7
+------------------
+
+CentOS 6 provides Python 2.6. However, OMERO.web requires Python 2.7
+in order to use `Django 1.8`. While `Django 1.6` may be used with Python
+2.6, this version of Django no longer has security support.  In
+consequence, it is necessary to upgrade to Python 2.7 in order to
+obtain Django security updates, which are required for a production
+deployment.
+This walkthrough assumes that packages like ``wget, zip, tar, pip`` are
```

Comment:
mainly to help @pwalczysko to start 
as discussed, removed and added to https://trello.com/c/QeAwP5Gz/104-installation-instructions-testing